contracts/anvil/deploy-helloworld.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
We have a PR to nixify the project.  It will be nice to change these to `#!/usr/bin/env bash`